README.md

```diff
@@ -392,6 +392,29 @@ The FTP user account is required when the Flexy's local time is not set to UTC.
+
+##### Configured Logging Level
+There are seven options for the configurable log level. The logging level is configured in the application configuration, detailed in Section 4. Each log level includes the output for the log levels below it (lower numerical value). For example, log level 3 (warning) includes the output for log level 2 (serious) and log level 1 (critical). All positive log levels print to the Flexy realtime logs, and negative log levels output for text files in the /usr directory of Ewon Flexy. Log text files are named logN.txt, where N is an integer.
+
+| LogLevel         | Description                                                                      |
+| :--------------: | :------------------------------------------------------------------------------- |
+| 6, -6 (Trace)    | Exception stack traces                                                           |
+| 5, -5 (Debug)    | Low level information about the state of the application                         |
+| 4, -4 (Info)     | Application state information                                                    |
+| 3, -3 (Warning)  | Issues encountered in the application that are not serious                       |
+| 2, -2 (Serious)  | Errors that are serious but recoverable                                          |
+| 1, -1 (Critical) | Critical application log messages (Startup, Initialization, Unrecoverable Error) |
+| 0 (None)         | Logging is disabled                                                              |
+
+##### Logging Performance
+The log output has an impact on the performance of the application. For normal operation, the log level should be set to the lowest numerical value (highest when outputting to log files) that will produce satisfactory logs. For debugging issues with the application or a device, higher numerical values (lower when outputting to log files) can be used to print additional information to help diagnose.
+
+Negative log values utilize log files in the /usr directory to store log output. This should only be enabled for short periods of time while diagnosing problems. Leaving this enabled for extended periods of time will cause excessive wear on the flash memory of the Ewon Flexy and could cause irreparable damage.
+
+##### Adding Log Output
+Log output can be added to the application by inserting calls to the logging class, Logger. Each log level has a method that will output the log to the appropriate location. For example, a call to `Logger.LOG_DEBUG(String)` will result in log output that is visible if the configured application log level is set to debug (5/-5) or trace (6/-6). A call to `Logger.LOG_CRITICAL(String)` will result in log output that is visible if the configured application log level is set to critical (1/-1) or a higher logging level. A call to `Logger.LOG_EXCEPTION(Exception)` will result in log output that is visible if the configured application log level is set to trace (6/-6).
+
 ### Development Environment
```

Comment:
In my opinion we should keep logging documentation in the logger library README and use links to direct users to that information. It would keep the documentation reusable in other projects.

Reply:
I agree although the logging library does not have this documented (the adding Logger.LOG_DEBUG, etc)

Reply:
Maybe any new information could get added to a PR for https://github.com/hms-networks/sc-flexy-logger-lib

Reply:
I think it is out of scope of this PR, but I think that a brief how-to summary should probably get added to the library repos in the future, as its mostly isolated to the Javadocs right now